sklego/common.py

```diff
@@ -166,3 +166,27 @@ def expanding_list(list_to_extent, return_type=list):
         return [listed]
 
     return [return_type(listed[: n + 1]) for n in range(len(listed))]
+
+
+def chunker(sequence, chunk_size, step_size):
```

Comment:
Wouldn't `sliding_window` be a better name for this?

Reply:
I also wouldn't mind a test for this, nothing fancy but can't hurt to have.

Reply:
Sure, I'll implement both suggestions!

Reply:
My colleague @davidelagano whom I worked with on `GroupTimeSeriesSplit` has implemented the suggestions!